src/components/Classes/components/ConfirmLeaveModal.js

```diff
@@ -3,24 +3,27 @@ import ReactModal from "react-modal";
 import { Container, Row, Col, Button } from "reactstrap";
 import * as fetch from "../../../lib/fetch.js";
 
-class ConfirmLeaveModal extends React.Component {
+function ConfirmLeaveModal(props) {
+  const [error, setError] = useState("");
```

Comment:
You need to import `useState` to use it here. For example, `import React, { useState } from "react";`